spec/views/posts/index.html.erb_spec.rb

```diff
@@ -1,23 +1,16 @@
 require 'rails_helper'
 
-RSpec.describe 'User Post Index Page', type: :feature do
-  before do
-    # Add setup code here if needed (e.g., create user and posts)
-  end
-
-  it 'displays user profile information and post details' do
-    visit user_posts_path(User.first)
-
-    expect(page).to have_content('Tom')
-    expect(page).to have_selector('img.user-image')
-    expect(page).to have_content('Number of Posts: 3')
-  end
-
-  it 'redirects to a post\'s show page when clicking on a post title' do
-    visit user_posts_path(User.first)
-
-    click_link('Hello')
+RSpec.feature 'Post Index Page', type: :feature do
+  scenario 'displays post information' do
+    user = User.create(name: 'Example User', bio: 'User Bio')
+    post = user.posts.create(title: 'Example Post', text: 'Post Text')
+    visit user_posts_path(user)
 
-    expect(current_path).to eq(post_path(Post.first))
+    expect(page).to have_content(user.name)
+    expect(page).to have_content(post.title)
+    expect(page).to have_content("Number of Posts: #{user.posts.count}")
+    expect(page).to have_content(post.text)
+    expect(page).to have_content("#{post.comments.count} Comments")
+    expect(page).to have_content("#{post.likes.count} Likes")
```

Comment:
- [ ] You are missing a test that asserts you can see a user's profile picture here.
![image](https://github.com/Gardimy/BlogApp/assets/45460192/5cdf1d95-6a11-4210-8d5e-7a907063ef6d)